### PR TITLE
Docs: Updated CRM Documentation

### PR DESCRIPTION
**Pull request by asbasawaraj**

### CODE REVIEW

File: erpnext_com/www/docs/user/manual/en/CRM/customer-group.md [hunk 1]
```diff
@@ -18,5 +27,21 @@ reports. An example of a sample report is given below:
...
+
+#### 2.1 Assign Credit Limit, Default Price List, and Default Payment Terms Template
+
+You can assign the credit limit, [Price List](/docs/user/manual/en/setting-up/price-lists), and [Payment Terms](/docs/user/manual/en/accounts/payment-terms) and they will be automatically applied when a customer belonging to the customer group is selected in sales transactions like [Sales Order](/docs/user/manual/en/selling/sales-order) and [Sales Invoice](/docs/user/manual/en/accounts/sales-invoice).
+
+#### 2.2 Default Receivable Account
+
+You need not create a separate accounting ledger for each customer in ERPNext. Read [Common Receivable Account](/docs/user/manual/en/accounts/articles/common-receivable-account) for more details.
+
+If you need a separate receivable account for a customer, you can add the same in 'DEFAULT RECEIVABLE ACCOUNT' section.
```
Comment (prasadarr) [inline erpnext_com/www/docs/user/manual/en/CRM/customer-group.md]: remove caps

File: erpnext_com/www/docs/user/manual/en/CRM/email_group.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
+<!-- add-breadcrumbs -->
+
+# Email Group
+
+**An Email Group is a list of subscribers email ids to which you send newsletters.**
```
Comment (prasadarr) [inline erpnext_com/www/docs/user/manual/en/CRM/email_group.md]: subscriber, remove the s

File: erpnext_com/www/docs/user/manual/en/CRM/email_group.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+You can manage subscribers by grouping them into different groups.
+
+You can group existing and potential customers into different groups based on their territory or products they are interested in. You can group the email id of all employees so that you can share information with them.
```
Comment (prasadarr) [inline erpnext_com/www/docs/user/manual/en/CRM/email_group.md]: email ID, ID caps everywhere

File: erpnext_com/www/docs/user/manual/en/CRM/opportunity.md [hunk 4]
```diff
@@ -59,11 +63,17 @@ Pipeline” this is what you need to track if you want to be able to predict how
 much business you are going to get in the future. Its always a good idea to be
 able to track what is coming in order to adjust your resources.
 
+#### 2.6 Capture the Reasons for Loosing Opportunities
+
+When an opportunity is lost, you can capture the reasons for loosing. This will help you to analyse the trends over a long period of time and identify the insights needed for improvements at various areas in the organisation.
```
Comment (prasadarr) [inline erpnext_com/www/docs/user/manual/en/CRM/opportunity.md]: losing with a single o

File: erpnext_com/www/docs/user/manual/en/CRM/email_group.md (hunk 2 above)
Comment (asbasawaraj) [reply erpnext_com/www/docs/user/manual/en/CRM/email_group.md]: I think its 'subscribers'

Comment (prasadarr) [reply erpnext_com/www/docs/user/manual/en/CRM/email_group.md]: no, subscriber.

**Pull request merged**